.github/workflows/main.yml

```diff
@@ -54,7 +54,8 @@ jobs:
         uses: docker/build-push-action@2eb1c1961a95fc15694676618e422e8ba1d63825 # pin@v3
         with:
           context: .
-          platforms: linux/amd64,linux/arm64,linux/ppc64le,linux/s390x
+          # platforms: linux/amd64,linux/arm64,linux/ppc64le,linux/s390x
+          platforms: linux/amd64,linux/arm64,linux/s390x
```

Comment:
who uses linux/s390x now a days ? :)

Reply:
you'd be surprised.... but IBM which is a major contributor to Istio is... :) 

dunno who uses ppc64le... hopefully nobody until they fix that error